backend/src/test/java/edu/alexu/cse/dripmeup/AdminServiceTest.java

```diff
@@ -1,83 +1,92 @@
 package edu.alexu.cse.dripmeup;
 
 import edu.alexu.cse.dripmeup.entity.AdminEntity;
-        import edu.alexu.cse.dripmeup.repository.AdminRepository;
-        import edu.alexu.cse.dripmeup.service.AdminService;
-        import org.junit.jupiter.api.BeforeEach;
-        import org.junit.jupiter.api.Test;
-        import org.mockito.InjectMocks;
-        import org.mockito.Mock;
-        import org.mockito.MockitoAnnotations;
+import edu.alexu.cse.dripmeup.repository.AdminRepository;
+import edu.alexu.cse.dripmeup.service.AdminService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.security.crypto.password.PasswordEncoder;
 
-        import static org.junit.jupiter.api.Assertions.assertFalse;
-        import static org.junit.jupiter.api.Assertions.assertTrue;
-        import static org.mockito.Mockito.when;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
 
-public class AdminServiceTest {
+class AdminServiceTest {
 
     @Mock
     private AdminRepository adminRepository;
 
+    @Mock
+    private PasswordEncoder passwordEncoder;
+
     @InjectMocks
     private AdminService adminService;
 
     @BeforeEach
-    public void setUp() {
+    void setUp() {
         MockitoAnnotations.openMocks(this);
     }
 
     @Test
-    public void testAdminLogin_Success() {
+    void testAdminLogin_Success() {
         String userName = "admin";
-        String password = "password";
+        String rawPassword = "password";
+        String encodedPassword = passwordEncoder.encode(rawPassword);
         AdminEntity admin = new AdminEntity();
         admin.setUserName(userName);
-        admin.setPassword(password);
+        admin.setPassword(encodedPassword);
 
         when(adminRepository.findByUserName(userName)).thenReturn(admin);
+        when(passwordEncoder.matches(rawPassword, encodedPassword)).thenReturn(true);
```

Comment:
> This line will invalidate the test, since the purpose of the test is to check that the method could find if raw Password matches with encoded Password. 

I have written this comment in UserService Test first time, so I mention it again.